freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/fix_next_update_grace_period.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+function setCaConfiguration() {
+    PROPERTY=$1
+    VALUE=$2
+    ESCAPED_PROPERTY=${PROPERTY//./[.]}
+    TMP_FILE=`mktemp`
+    cat /var/lib/pki/pki-tomcat/ca/conf/CS.cfg | sed "s/.*$ESCAPED_PROPERTY=.*/$PROPERTY=$VALUE/g" > $TMP_FILE
```

Comment:
this could be replaced with `sed -i`, so you don't need a temp file

Reply:
This entire script was replaced with the salt `file.replace`.